imports/plugins/core/revisions/client/components/publishControls.js

```diff
@@ -132,7 +132,7 @@ class PublishControls extends Component {
 
   get isVisible() {
     if (Array.isArray(this.props.revisions) && this.props.revisions.length && this.primaryRevision) {
-      const primaryRevision = this.primaryRevision;
+      const primaryRevision = this.primaryRevisionObj;
```

Comment:
`const primaryRevision = this.primaryRevision;`
eslint sees this as something that should use object destructuring, but it's not, since this.primaryRevision is pointing to that method on the class.
Hence changing the method name

Reply:
It would be more backward compatible to change the variable name rather than the getter method name.